### DISCUSSION TITLE
Why are the static images not in the output directory after packaging?

### DISCUSSION THREAD

**Discussion by Hyebin98** [wxt-dev/wxt#1303] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/5f2a1da7-6bc1-427c-a2fb-c519c3dad1e5)
I am a beginner who has just started with browser extension development. What should I do?

**Comment by aiktb** [comment]
Use `public` directory

**Comment by aklinker1** [comment]
https://wxt.dev/guide/essentials/assets.html

**Reply by Hyebin98** [reply, marked as answer]
![image](https://github.com/user-attachments/assets/4b1319eb-f005-4ba8-aa13-bcc1e1412236)
I solved it using moudle
